systemtests/publisher.test.ts

```diff
@@ -96,22 +99,24 @@ describe('RabbitMQ publisher', () => {
 
 			expect(response.data.length).to.eql(1);
 			expect(response.data[0].message_count).to.eql(0);
-			expect(response.data[0].payload).to.eql('OnsidHlwZSI6InJlY29yZCIsImZpZWxkcyI6W119AJABeyJuYW1lIjoidGVzdCIsInR5cGUiOiJyZWNvcmQiLCJmaWVsZHMiOlt7Im5hbWUiOiJpZCIsInR5cGUiOiJzdHJpbmcifV19Dgx0ZXN0SWQ=');
+			expect(response.data[0].payload).to.eql(expectedPayload, `\nactual ${Buffer.from(expectedPayload, 'base64').toString()}\nexpected ${Buffer.from(response.data[0].payload, 'base64').toString()}\nraw`);
```

Comment:
OPTIONAL: Consider extracting to a function `stringifyPayload(..)`, as it's reused all over your tests.